cuisino/evaluation/views.py

```diff
@@ -0,0 +1,49 @@
+from django.shortcuts import render
+from evaluation.models import Evaluation, Answer
+
+
```

Comment:
this coding should be affected by the changes made in models/forms. (make adjustments to this after you make the changes to the other files.)

Reply:
perhaps from showresult function, we can have it affect the other models in other applications. Dont forget, for each complement/complaint, it will promote/demote the employee.